packages/paste-core/components/pagination/src/types.ts

```diff
@@ -0,0 +1,46 @@
+export type AsTags = 'a' | 'button';
+export type Variants = 'back' | 'forward';
+
+export interface PaginationProps extends React.HTMLAttributes<'nav'> {
+  label: string;
+  children: React.ReactNode;
+}
+
+export interface PaginationItemsProps extends React.HTMLAttributes<'ul'> {
+  children: React.ReactNode;
+}
+
+export interface PaginationLabelProps extends React.HTMLAttributes<'div'> {
+  children: React.ReactNode;
+}
+
+export interface PaginationButtonProps extends React.HTMLAttributes<'button'> {
+  as?: AsTags;
+  href?: string;
+}
+
+export interface PaginationArrowProps extends PaginationButtonProps {
+  label: string;
+  variant?: Variants;
```

Comment:
I think variant should be a required prop for Arrow, it's weird omitting it and makes code a little confusing as I wasn't sure what an arrow without `back/forward` meant without digging in the code.

Reply:
Good point. Thats also inline with our Button component.